FauxPasConfig/main.fauxpas.json

```diff
@@ -1,7 +1,7 @@
 // Generated from the Faux Pas GUI
 {
     // Xcode target to check (String)
-    "target": "StripeiOSStatic",
+    "target": "StripeiOS",
```

Comment:
Yeah - I think FP's checker is better for framework targets in general. The only reason we were formerly defaulting to StripeiOSStatic is that it had a minimum iOS target of 7.0, which was useful for APIAvailability stuff. Now that we only support 8.0, I think this makes more sense as the default thing to check. (And the CI script will still check both)